free-courses-en.md

```diff
@@ -79,6 +79,9 @@
 
 * [Bash tutorial](http://gdrcorelec.ups-tlse.fr/files/bash.pdf) (PDF)
 * [Bento Shell Track](https://bento.io/topic/shell) (Bento)
+* [Linux Shell Scripting Tutorial v1.05r3](http://www.freeos.com/guides/lsst/) 
```

Comment:
seems old. what about https://bash.cyberciti.biz/guide/Main_Page Also, is it a course or is it a book?